src/shared/components/main-menu/main-menu.component.ts

```diff
@@ -65,29 +65,92 @@ export class MainMenuComponent {
    * @event
    */
   @Output() beginButtonClick = new EventEmitter<void>();
+  @Output() buttonOneClick = new EventEmitter<void>();
+  @Output() buttonTwoClick = new EventEmitter<void>();
+  @Output() buttonThreeClick = new EventEmitter<void>();
+
+  /**
+   * Flag que controla o estado de abertura/fechamento do menu.
+   *
+   * @type {boolean}
+   * @default false
+   */
+  isMenuOpen = false;
+
+  /**
+   * Flag que indica se a aplicação está sendo exibida em um dispositivo móvel.
+   *
+   * @type {boolean}
+   * @default false
+   */
+  isMobile = false;
 
   /**
    * Manipula o evento de clique do botão inicial.
    * Emite um evento através do beginButtonClick.
+   * @event {void} beginButtonClick - Evento emitido quando o botão inicial é clicado.
    */
   onBeginClick() {
     this.beginButtonClick.emit();
-    isMenuOpen = false;
-    isMobile = false;
+  }
+
+  /**
+   * Manipula o evento de clique do primeiro botão do menu.
+   * Emite um evento através do buttonOneClick.
+   * @event {void} buttonOneClick - Evento emitido quando o primeiro botão é clicado.
+   */
+  onButtonOneClick(): void {
+    this.buttonOneClick.emit();
+  }
 
+  /**
+   * Manipula o evento de clique do segundo botão do menu.
+   * Emite um evento através do buttonTwoClick.
+   * @event {void} buttonTowClick - Evento emitido quando o segundo botão é clicado.
+   */
+  onButtonTwoClick(): void {
+    this.buttonTwoClick.emit();
+  }
+
+  /**
+   * Manipula o evento de clique do terceiro botão do menu.
+   * Emite um evento através do buttonThreeClick.
+   * @event {void} buttonThreeClick - Evento emitido quando o terceiro botão é clicado.
+   */
+  onButtonThreeClick(): void {
+    this.buttonThreeClick.emit();
+  }
+
+  /**
+   * Construtor do componente MainMenu.
+   * Inicializa o estado do componente e adiciona um listener para redimensionamento da janela.
+   */
   constructor() {
     this.checkScreenSize();
     window.addEventListener('resize', () => this.checkScreenSize());
   }
 
+  /**
+   * Verifica o tamanho da tela e atualiza a flag isMobile.
+   * Define isMobile como true se a largura da janela for menor ou igual a 768 pixels.
+   * @returns {void}
+   */
   checkScreenSize() {
```

Comment:
Qual a necessidade disso? Se for só pra estilização, aplique breakpoints no scss